src/main/groovy/de/dkfz/eilslabs/tools/execution/ExecutionLevel.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
This file directly relates to ExecutionContext in the Roddy core project. Please move it back.